style.css

```diff
@@ -0,0 +1,199 @@
+nav, footer {
```

Comment:
More a wondering that a concern: you did not use any IDs in your CSS as attributes - was there a reason for this?  On review of your pages, are there any places that may warrant use of IDs over classes?  It might be interesting to look into the differences between them, and the way in which IDs also have an ability to hold location on a page.  Having said all of this, it is quite possible to have good html/css without the use of IDs, hence this being more of a thought...